en/mkdocs.yml

```diff
@@ -64,6 +64,7 @@ nav:
     - Observability:
           - Observability Overview: observability/observability-overview.md
           - Perform Root Cause Analysis: observability/perform-root-cause-analysis.md
+          - Performance Analysis: references/analyze-performance.md
```

Comment:
Can we have a separate section (like `Observability`) for performance analyzer? Because performance analyzer conducts forecast of application performance while observability provides what actually the applications' performance behavior was. IMO having it under the observability may create wrong impression on the user that we do analyze any application's actual performance.